app/helpers/presenters/publishing_api/worldwide_office_helper.rb

```diff
@@ -0,0 +1,36 @@
+module Presenters::PublishingApi::WorldwideOfficeHelper
+  def worldwide_office_details(worldwide_office)
+    {
+      access_and_opening_times: access_and_opening_times(worldwide_office),
+      services: services(worldwide_office),
+      type: worldwide_office.worldwide_office_type.name,
+    }
+  end
+
+  def worldwide_office_parts(worldwide_offices)
+    worldwide_offices.map do |worldwide_office|
+      worldwide_office_details(worldwide_office).merge(
+        contact_content_id: worldwide_office.contact.content_id,
+        slug: "office/#{worldwide_office.slug}",
```

Comment:
Is there any way we could DRY this up to avoid hardcoding it here. 

Maybe raising the path as a constant in `WorldwideOffice`, or something?

Reply:
I don't know what the chance of someone changing the `office` path is, but if they did, then this would break

Reply:
Good point.  I've used the `base_path` method of the `WorldwideOffice`, then gsubbed out the base path of the `WorldwideOrganisation` and a slash.